lisa/tools/perf.py

```diff
@@ -31,12 +31,9 @@ def _install(self) -> bool:
             kernel_ver = (
                 self.node.tools[Uname].get_linux_information().kernel_version_raw
             )
-            if (
-                isinstance(self.node.os, Redhat)
-                or isinstance(self.node.os, CentOs)
-                or isinstance(self.node.os, CBLMariner)
-                or isinstance(self.node.os, Suse)
-            ):
+            if isinstance(self.node.os, CBLMariner):
+                self.node.os.install_packages("kernel-tools")
+            elif isinstance(self.node.os, (CentOs, Redhat, Suse)):
```

Comment:
Does it work like this way? CentOs is from Redhat, so it doesn't need to be in the list.

Reply:
yes, worked, and I have removed CentOS.